servers/rendering/renderer_rd/effects/ss_effects.cpp

```diff
@@ -668,8 +668,14 @@ void SSEffects::screen_space_indirect_lighting(Ref<RenderSceneBuffersRD> p_rende
 		ssil.gather_push_constant.screen_size[0] = p_settings.full_screen_size.x;
 		ssil.gather_push_constant.screen_size[1] = p_settings.full_screen_size.y;
 
-		ssil.gather_push_constant.half_screen_pixel_size[0] = 1.0 / p_ssil_buffers.buffer_width;
-		ssil.gather_push_constant.half_screen_pixel_size[1] = 1.0 / p_ssil_buffers.buffer_height;
+		ssil.gather_push_constant.half_screen_pixel_size[0] = 2.0 / p_settings.full_screen_size.x;
```

Comment:
I think `buffer_width` and `buffer_height` are already half `full_screen_size.xy`, Seems to be we should use `1.0/buffer_width` for normal and '1.0/half_width` for `ssil_half_size`?

Reply:
I might be wrong though because `buffer_width` I think is also halved if `ssil_half_size`

Reply:
The problem is 1.0 / buffer_width does not equal 2.0 / screen_width

For example. If the screen is 1001 pixels wide, 2.0/ screen_width is ``0.001998001998`` while 1.0 / buffer width is ``0.001996007984`` that small difference is enough to cause an artifact when sampling the entire screen which is exactly what we see in the attached issues

Reply:
Hmm, interesting, so its the source size that is more important than the size with which we create the destination texture width.